demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -248,8 +248,16 @@ def generate_single_command_section(cmd: dict, example_dict: dict, command_permi
             if not arg.get('description'):
                 errors.append(
                     'Error! You are missing description in input {} of command {}'.format(arg['name'], cmd['name']))
+            if not arg.get('description').endswith('.'):
```

Comment:
no need to read from the dictionary 3 times and assign to it
just init a variable with it and work with that instead